### PR TITLE
Add simple SafeSnap setup instructions.

### PR DESCRIPTION
**Pull request by auryn-macmillan**

### CODE REVIEW

File: docs/SafeSnapSetup.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Setup a SafeSnap DAO
+
+1. Deploy your [Gnosis Safe](https://gnosis-safe.io).
+
+2. Modify the question template at `src/tasks/defaultTemplate.json` to suit your DAO.
+
+3. `yarn build`
+
+4. `yarn hardhat --network <network> createDaoTemplate --oracle <oracle_address>`
```
Comment (rmeissner) [inline docs/SafeSnapSetup.md:9]: Should we link where you can find the oracle addresses?

File: docs/SafeSnapSetup.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Setup a SafeSnap DAO
```
Comment (rmeissner) [inline docs/SafeSnapSetup.md:1]: Should we link this file in the readme?

File: docs/SafeSnapSetup.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+1. Deploy your [Gnosis Safe](https://gnosis-safe.io).
+
+2. Modify the question template at `src/tasks/defaultTemplate.json` to suit your DAO.
+
+3. `yarn build`
+
+4. `yarn hardhat --network <network> createDaoTemplate --oracle <oracle_address>`
+
+5. `yarn hardhat --network <network> setup --dao <safe_address> --oracle <realitio_address> --timeout 172800 --cooldown 86400`
```
Comment (rmeissner) [inline docs/SafeSnapSetup.md:11]: template id is missing here

**Comment by auryn-macmillan** [comment]
I think we can close this one since #17 is much more comprehensive.

**Pull request closed without merging**